frontends/web/src/routes/account/walletconnect/index.tsx

```diff
@@ -1,473 +0,0 @@
-        }
-        return;
-      default:
-        console.log('not supported');
-      }
-    },
-    [code, handleSessionRequest]
-  );
-
-  const onSessionDelete = useCallback(
-    (deleteRequest: SignClientTypes.EventArguments['session_delete']) => {
-      const { topic } = deleteRequest;
-      setSessions(sessions => sessions.filter(session => session.topic !== topic));
-    },
-    []
-  );
-
-  async function disconnect(topic: string) {
-    await web3wallet.disconnectSession({
-      topic,
-      reason: getSdkError('USER_DISCONNECTED'),
-    });
-    updateSessions();
-  }
-
-  const onApprove = useCallback(async () => {
-    if (currentProposal && currentAddresses) {
-      const { id, params } = currentProposal;
-      const { requiredNamespaces } = params;
-      let accounts: string[] = [];
-      Object.keys(requiredNamespaces).forEach(key => {
-        requiredNamespaces[key].chains?.map(chain => accounts.push(`${chain}:${currentAddresses[0].address}`));
-      });
-
-      // ------- namespaces builder util ------------ //
-      const approvedNamespaces = buildApprovedNamespaces({
-        proposal: params,
-        supportedNamespaces: {
-          eip155: {
-            chains: SUPPORTED_CHAINS,
-            methods: ['eth_sendTransaction', 'eth_signTransaction', 'eth_sign', 'personal_sign', 'eth_signTypedData', 'eth_signTypedData_v4'],
-            // TODO: handle emitting accountsChanged events
-            // TODO: handle emitting chainChanged event, we need to have a chain selector in the app to support other networks in dapps properly
-            events: ['accountsChanged', 'chainChanged'],
-            accounts: accounts
-          },
-        },
-      });
-
-      await web3wallet?.approveSession({
-        id,
-        namespaces: approvedNamespaces
-      });
-
-      setApproveModal(false);
-      setUri('');
-      setCurrentProposal(undefined);
-      updateSessions();
-    }
-  }, [currentAddresses, currentProposal]);
-
-  async function onReject() {
-    if (currentProposal) {
-      await web3wallet.rejectSession({
-        id: currentProposal.id,
-        reason: getSdkError('USER_REJECTED_METHODS')
-      });
-    }
-    setApproveModal(false);
-    setUri('');
-    setCurrentProposal(undefined);
-  }
-
-  async function onConnect(uri: string) {
-    try {
-      await pair({ uri });
-    } catch (err: unknown) {
-      alert(err);
-    } finally {
-      setUri('');
-    }
-  }
-
-  useEsc(() => route(`/account/${code}`));
-
-  useEffect(() => {
-    if (receiveAddresses) {
-      setCurrentAddresses(receiveAddresses[currentAddressIndex].addresses);
-    }
-  }, [receiveAddresses, currentAddresses]);
-
-
-  useEffect(() => {
-    web3wallet?.on('session_proposal', onSessionProposal);
-    web3wallet?.on('session_request', onSessionRequest);
-    web3wallet?.on('session_delete', onSessionDelete);
-    web3wallet?.on('auth_request', onAuthRequest);
```

Comment:
before POC is gone, do we need `auth_request`? 

also is there anything else in the POC that has not yet been implemented or I should check? please double check if there was anything else in this POC we need. Maybe ping Tom if it makes sense for him to take a look.